Include/cpython/bytesobject.h

```diff
@@ -14,6 +17,13 @@ typedef struct {
      */
 } PyBytesObject;
 
+#define PyBytes_GET_BSTATE(op)       (((PyBytesObject *)(op))->bstate)
+#define PyBytes_SET_BSTATE(op, val)  (((PyBytesObject *)(op))->bstate = (unsigned int)(val))
+
+#define PG_BSTATE_LOAD_BYTES(op_) \
```

Comment:
I'm not really sure what the intention of this macro is, and the one call site doesn't immediately make it obvious to me. Can we provide a quick doc string?